client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -135,6 +136,8 @@ function mediaButton( editor ) {
 		return { isLoaded, onLoad };
 	} )();
 
+	const mediaHasChanged = ( media ) => withoutHttp( media.guid ) !== withoutHttp( media.URL );
```

Comment:
Once the media is edited for the first time it always will return `true`.

I've added a console and just in some `initial` states its value is `true`.

![image](https://cloud.githubusercontent.com/assets/77539/19967637/fd05e89e-a1af-11e6-82c3-04d4d3d2bb37.png)

Also, I've edited the code returning always `true` and I don't see any change in terms of the functionality.

![image](https://cloud.githubusercontent.com/assets/77539/19967677/24e581c6-a1b0-11e6-8cd7-d42cb235b707.png)

And btw, it works!. So I wonder if we should just remove this helper function.